staging/prometheus-operator/values.yaml

```diff
@@ -1611,7 +1611,7 @@ prometheus:
     ##
     image:
       repository: quay.io/prometheus/prometheus
-      tag: v2.18.2
+      tag: v2.20.0
```

Comment:
I suggest we bump it to 2.21.0 which is now available.

Reply:
it could already break with switching to `v2.20.0`
the prometheus-operator in the version we use doesn't have it in the compatibility list
https://github.com/prometheus-operator/prometheus-operator/blob/v0.38.1/Documentation/compatibility.md#prometheus

Reply:
We need to bump the operator to support this new version.

Reply:
We have a ticket to spike out upgrading prometheus-operator to using the new chart https://jira.d2iq.com/browse/D2IQ-73452 (anybody is free to work on it ;) )